microsoft-edge/webview2/release-notes.md

```diff
@@ -24,7 +24,52 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
+
+### General  
+*   WebView2 performance improvements.
+*   Reliability fixes. \([\#1605][GithubMicrosoftedgeWebviewfeedbackIssue1605] and [\#1678][GithubMicrosoftedgeWebviewfeedbackIssue1678]\)
+*   Added performance improvements during startup and when the host app is in the foreground.
+
+#### Experimental Features  
+*   Removed silent failures by using `EnsureCoreWebView2Async`, which throws an `ArgumentException` when called multiple times with incompatible parameters.
+*   Changed default handling of the [UserDataFolder][Webview2ReferenceWin32Icorewebview2experimentalenvironment5ViewWebview210999PrereleaseGetUserdatafolder] property in the environment object. 
+    > [!CAUTION]
+    > **Breaking Change**:  The default handling for the UDF if the developer doesn't specify where to put it will be changing. For details, navigate to [Announcement: User directory folder default handling updates][WebView2UDFAnnouncement].
+*   Added [navigation & script APIs][Webview2ReferenceWin32Icorewebview2experimentalframeViewWebview210999Prerelease] for iframes. 
+*   Added `MemoryUsageTargetLevel` which allows developers to specify memory consumption levels, such as low, or normal. 
+*   Added `ExclusiveUserDataFolderAccess` to environment options.
+*   Added `HiddenPdfToolbarItems` to customize PDF toolbar items.
+*   Added `PrintToPdf`, which allows printing the current page to PDF. Also, you can use optional custom settings with this new API.
+*   Added `AllowExternalDrop` property to allow the dragging and dropping of objects from outside a WebView2 control into it. 
+*   Added ContextMenu APIs which allow customization of the WebView2 context menu. 
+
+#### Bug fixes  
+*   Improved how host objects exceptions are caught in your JavaScript code.
+*   Replaced WebView2 icon with a generic icon in DevTools windows.
+*   Turn on the Tab screen sharing option when `MediaDevices.getDisplayMedia()` is used. \([\#1566][GithubMicrosoftedgeWebviewfeedbackIssue1566]\)
+*   Fixed a bug in the Client Certificate API, when the correct certificate was not selected. This is a Runtime change. \([\#1666][GithubMicrosoftedgeWebviewfeedbackIssue1666]\)
+*   Fixed bug where `window.chrome.webview` was unavailable in new windows in the same parent domain. This change is Runtime-specific. \([\#1144][GithubMicrosoftedgeWebviewfeedbackIssue1144]\)
+*   Fixed a bug where dropdown menus or lists were displayed behind the window that has focus. \([\#411][GithubMicrosoftedgeWebviewfeedbackIssue411]\)
+*   Fixed focus issues when using `put_IsVisible(false)`. \([\#238][GithubMicrosoftedgeWebviewfeedbackIssue238]\)
+*   Fixed a bug to apply `SetVirtualHostNameToFolderMapping` to popup windows.
+*   Fixed bugs where an `IDispatch` objects were returned as `IUnknown`.
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:
+*   `IsSwipeNavigationEnabled`.
+*   `BrowserProcessExited`.
+*   `OpenBrowserTaskManager`.
 
 ## 1.0.961.33
```

Comment:
Can we change the minimum version for 1.0.961.33 package to 93.0.961.**44**, the minimum version that includes a runtime client certificate api change?

Reply:
I managed to add that comment as a Suggestion on line 80